lib/crystalball.rb

```diff
@@ -14,6 +14,7 @@
 require 'crystalball/map_generator/configuration'
 require 'crystalball/map_generator/coverage_strategy'
 require 'crystalball/map_generator/allocated_objects_strategy'
+require 'crystalball/map_generator/parser_strategy'
```

Comment:
I think parser strategy should be included separately in favor of https://github.com/toptal/crystalball/pull/66/files#r185799760